.github/workflows/packages.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           path: dbld/build/*.tar.gz
 
   packages:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
There are some `ubuntu-18.04` in devshell, I think those can be changed for the same reason.